apps/next/src/memlab/config/thresholds.json

```
@@ -0,0 +1,78 @@
+{
+  "global": {
+    "maxLeakedObjects": 0,
+    "maxRetainedSize": 1048576,
+    "notes": "Default: zero tolerance for leaks, max 1MB retained"
+  },
+  "components": {
+    "drawer": {
+      "maxLeakedObjects": 2,
+      "maxRetainedSize": 524288,
+      "notes": "Drawer uses portal and ViewStack - may have minor DOM cleanup delays",
+      "knownIssues": []
+    },
+    "button": {
+      "maxLeakedObjects": 0,
+      "maxRetainedSize": 10240,
+      "notes": "Simple component with zero tolerance for leaks"
+    },
+    "tooltip": {
+      "maxLeakedObjects": 2,
+      "maxRetainedSize": 102400,
+      "notes": "Uses React Aria's overlay system and portal - may have minor leaks from floating positioning during stress tests"
+    },
+    "dialog": {
+      "maxLeakedObjects": 2,
+      "maxRetainedSize": 262144,
+      "notes": "Modal overlay with portal rendering - uses PortalProvider and ModalOverlay"
+    },
+    "notice": {
+      "maxLeakedObjects": 3,
+      "maxRetainedSize": 524288,
+      "notes": "Uses @accelint/bus for event subscriptions and toast queue - critical to verify subscription cleanup"
+    },
+    "intentional-leak": {
+      "maxLeakedObjects": 100,
+      "maxRetainedSize": 10485760,
+      "notes": "Intentional leak test page - high thresholds to allow leak detection verification. Tests validate that MemLab correctly detects memory leaks.",
+      "knownIssues": [
+        "Intentional DOM element leaks",
+        "Intentional closure leaks"
+      ]
+    },
+    "accordion": {
+      "maxLeakedObjects": 0,
+      "maxRetainedSize": 20480,
+      "notes": "Simple expand/collapse component - zero tolerance for leaks"
+    },
+    "accordiongroup": {
```

nit: could be more consistent here

```suggestion
    "accordion-group": {
```